inc/namespace.php

```diff
@@ -5,14 +5,40 @@
 use Exception;
 use function HM\Platform\get_aws_sdk;
 
-/**
- * Bootstrapper for the plugin.
+/*
+ * Set initial values and register handlers
+ */
+function initialize() {
+	$GLOBALS['hm_platform_xray_errors'] = [];
```

Comment:
@nathanielks where did this code come from? Was this from the main `hm-platform` load.php? If so, do we need to coordinate an update there too?

Reply:
Aye, coordination will be required. See this comment: https://github.com/humanmade/aws-xray/pull/22#issuecomment-453347542